activities/staff-management/sickness.md

```diff
@@ -32,9 +32,8 @@ Let <hr@publiccode.net> know when you resume your regular work.
 
 Please do this the afternoon before you plan to come back to work. Or when you are well again, even if the next day is not a work day for you.
 
-You can always call again the next day if you were too optimistic (and need to stay home a bit longer).
+You can always check in again the next day if you were too optimistic (and need to stay home a bit longer).
 
 Alternatively, email the morning (when you resume work), at the latest.
 
-This is important as HR needs to end the notification of your sickness in the HR portal.
-This enables us to know you are back in and avoids action by the arbo-dienst.
+This is important as HR needs to end the notification of your sickness in the HR portal. This enables us to know you are back in and avoids action by the arbo-dienst.
```

Comment:
@demkodo, this is an uneccesary change as a single newline will not render as such in MarkDown, only a empty newline will create a new paragraph. A new line for do make it easier to track changes in Git, so I'll change this back before merging :).

```suggestion
This is important as HR needs to end the notification of your sickness in the HR portal.
This enables us to know you are back in and avoids action by the arbo-dienst.
```